### PR TITLE
Readme.md updates

### PR DESCRIPTION
**Pull request by TRnonodename**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,8 +1,27 @@
 # Introduction #
-CM-Well is a writable Linked Data repository, developed by Thomson Reuters and used as its central Knowledge Graph database. CM-Well (Content Matrix Well) adheres to Open Data principles, meaning that its data is in a standard, machine-readable format.
+CM-Well is a writable Linked Data repository, developed by Thomson Reuters and used as its central Knowledge Graph database. CM-Well (Content Matrix Well) adheres to RDF  principles, meaning that its data is in a [standard](https://www.w3.org/RDF/), machine-readable format.
```
Comment (matankdr) [inline Readme.md]: there is an extra space character after RDF, I think it might be removed :)

**Comment by TRnonodename** [comment]
Done

**Comment by TRnonodename** [comment]
Trying again!

**Comment by matankdr** [comment]
Done :)

Matan Keidar
Technology & Ops - Platform
Thomson Reuters
Phone: +972-37350086
matan.keidar@tr.com<mailto:matan.keidar@tr.com>





On 19 Jul 2017, at 16:24, Dan Bennett <notifications@github.com<mailto:notifications@github.com>> wrote:


Trying again!

**Pull request merged**